capstone/src/main/java/com/google/sps/servlets/utils/ServletUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+
+public final class ServletUtil {
+  public static ImmutableList<String> getPropertyList(Entity entity, String property) {
```

Comment:
Let's add a test for this method, even if it's really simple.

Reply:
Done! Querying from Datastore worked :)